gatorgrade/main.py

```diff
@@ -2,6 +2,7 @@
 from pathlib import Path
 import typer
 
+
```

Comment:
You shouldn't need to modify this file -- it passes `black` in `main`, and it's best to keep your changes concentrated.
```suggestion
```

Reply:
Good find! I'm not sure how that happened but I fixed it in my latest push. Thanks!